### DISCUSSION TITLE
Finding my account where it isnt registered?

### DISCUSSION THREAD

**Discussion by MiguelCoelho27** [sherlock-project/sherlock#2031] [Q&A]
Hi everyone, im running the command 
python3 sherlock x@email.com --print-found for example, and it always returns me 14 websites, the same 14 websites even if i put another username, which are:
CGtrader, Chaos, Football, Slides, trello, authorSTREAM, chaos.social, igromania, jeuxvideo, mastodon.social, mastodon.xyz, mstdn.io, opennet and social.tchncs.de.
I've checked, and that email isnt registered in any of this websites, so can anyone explain me if this is some bug from Sherlock, or what does this means please?

Thanks :D

**Comment by ppfeister** [comment]
Hey @MiguelCoelho27 

CGtrader, Slides, and jeuxvideo are being addressed in #2068.

It appears that a few of them use various bot detection mechanisms or other WAFs, which would be suppressed by #2069. If the site uses either Cloudflare or PerimeterX, a fingerprint has been added for it and it will no longer present false pos after merge. If it uses Fastly or some other WAF and you would like to aide in fingerprinting it, just lmk and we can arrange to add another fingerprint before merge.

Beyond that, are you able to tell if any of those sites, aside from the ones I listed here, are both ( 1 ) still giving you false positives and ( 2 ) not behind a WAF?

___

**Edit:** Both patches have been merged.
